src/sunpy_sphinx_theme/sunpy/templates/sections/footer-content.html

```diff
@@ -0,0 +1,39 @@
+<div class="footer-page row fixed-bottom">
```

Comment:
Huhhhh, putting this in here actually worked?! Madness!

We can just drop layout.html completely then.